cmd/convox/start.go

```diff
@@ -23,6 +23,12 @@ func init() {
 		Description: "start an app for local development",
 		Usage:       "[directory]",
 		Action:      cmdStart,
+		Flags: []cli.Flag{
+			cli.StringFlag{
```

Comment:
you can specify a deafult value in this block so you don't need to check it against "" later